flutter/shell/platform/tizen/flutter_tizen_engine.cc

```diff
@@ -376,10 +371,9 @@ void FlutterTizenEngine::SetupLocales() {
   // Convert the locale list to the locale pointer list that must be provided.
   std::vector<const FlutterLocale*> flutter_locale_list;
   flutter_locale_list.reserve(flutter_locales.size());
-  std::transform(
-      flutter_locales.begin(), flutter_locales.end(),
-      std::back_inserter(flutter_locale_list),
-      [](const auto& arg) -> const auto* { return &arg; });
+  std::transform(flutter_locales.begin(), flutter_locales.end(),
+                 std::back_inserter(flutter_locale_list),
+                 [](const auto& arg) -> const auto* { return &arg; });
```

Comment:
The latest VS Code C++ extension uses clang-format-16 by default as I mentioned earlier in the messenger. You should manually specify the clang-format path with `/usr/bin/clang-format-15` in the VS Code settings. This will be fixed in the future.